contrib/CMakeLists.txt

```diff
@@ -115,12 +115,25 @@ endif()
 add_contrib (llvm-project-cmake llvm-project)
 add_contrib (libfuzzer-cmake llvm-project)
 add_contrib (libxml2-cmake libxml2)
-add_contrib (aws-s3-cmake
+
+add_contrib (aws-cmake
     aws
+    aws-c-auth
+    aws-c-cal
```

Comment:
Why do we need so many modules now? How has it worked before without them?

Reply:
It worked before it was before AWS SDK team added their own submodule system, see
https://github.com/aws/aws-sdk-cpp/commit/bb964685686c211d078fba7b8c840b73e16430de